layouts/error.vue

```diff
@@ -32,6 +9,12 @@ export default {
 			type: Object,
 			default: () => {}
 		}
+	},
+	data () {
+		return {
+			title: (this.error.statusCode === 404) ? "Page not found" : "An error occurred",
+			content: (this.error.statusCode === 404) ? "<p>The page you were looking for could not be found. It might have<br>been removed, renamed, or did not exist in the first place.</p><div><p><nuxt-link to='/'>Home page</nuxt-link></p></div>" : "<div><p><nuxt-link to='/'>Home page</nuxt-link></p></div>"
```

Comment:
A few questions:

- Why is there a `<br>` in the middle of the sentence?
- Why is there a `<div>` around some of the paragraphs?

Reply:
* It is a long sentence and felt it deserved a line break
* There are divs to serparate `<div><p><nuxt-link to='/'>Home page</nuxt-link></p></div>` (the linke leading back to the home page) from the rest of the content

Reply:
@TeddTech:

>     * It is a long sentence and felt it deserved a line break

On different sized screens, this won't work. For example on a small phone, it'll look like this:

<img width="319" alt="Screen Shot 2020-02-19 at 11 14 39 AM" src="https://user-images.githubusercontent.com/605361/74862264-14fa9900-5309-11ea-9808-6ba2389788f5.png">

It's better to use CSS to set the container width appropriately rather than forcing line breaks. Please remove the `<br />` tag.

>     * There are divs to serparate `<div><p><nuxt-link to='/'>Home page</nuxt-link></p></div>` (the linke leading back to the home page) from the rest of the content

Take a look at this: https://codepen.io/greatislander/full/XWbjXgM

There's no need for the extra `<div>`; `<p>` tags are block elements and provide the necessary separation.

Reply:
Okay, these changes have now been applied.